.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,55 @@
+name: Create Prerelease
+
+on:
+  workflow_dispatch:
+
+jobs: 
+  create_prerelease:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+    - name: Install Ruby 3.2
+      uses: ruby/setup-ruby@7d546f4868fb108ed378764d873683f920672ae2 # tag v1.149.0
+      with:
+        ruby-version: 3.2
+
+    - name: Checkout code
+      uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # tag v3.5.0
+
+    - run: bundle
+
+    - name: Update files with new version
+      run: bundle exec rake newrelic:version:bump
+
+    - name: Update newrelic.yml (TODO)
+      run: echo "after hannahs work is merged"
+
+    - name: Set tag name
+      run: echo "prerelease_tag=$(bundle exec rake newrelic:version:current)-pre" >> $GITHUB_ENV
+
+    - name: Create pull request
+      run: |
+        git checkout -b "prerelease_updates_${{ env.prerelease_tag }}"
+        git add --all
+        git config --global user.email "ruby-agent@newrelic.com"
+        git config --global user.name "Github bot"
+        git commit -m "bump version"
+        git push --set-upstream origin "prerelease_updates_${{ env.prerelease_tag }}"
+        gh pr create --label $LABEL --title "$TITLE" --body "$BODY"
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        TITLE: "Prerelease #{{env.prerelease_tag}}"
+        BODY: "Updates the version number, changelog, and newrelic.yml(if it needs updating). This is an automated PR"
+        LABEL: prerelease
+       
+    - name: Create pre release tag
+      uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 # tag v0.1.15
+      with:
+        tag_name: ${{ env.prerelease_tag }}
+        name: ${{ env.prerelease_tag }}
+        target_commitish: prerelease_updates_${{ env.prerelease_tag }}
```

Comment:
"commitish" lol

Reply:
lmao i know right, i also had never seen that before